build.go

```diff
@@ -18,7 +18,7 @@ import (
 	"github.com/openSUSE/umoci/pkg/fseval"
 	ispec "github.com/opencontainers/image-spec/specs-go/v1"
 	"github.com/pkg/errors"
-	"github.com/vbatts/go-mtree"
+	mtree "github.com/vbatts/go-mtree"
```

Comment:
The package is actually called 'mtree'; what is your editor doing here? If we're going to do this, we should do it tree-wide, but it seems like it shouldn't be necessary.